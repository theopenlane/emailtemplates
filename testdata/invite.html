{{ template "base.html" . }}

{{- define "content" -}}
custom invite email template
{{- end -}}