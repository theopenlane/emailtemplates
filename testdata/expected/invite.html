<div class="header">
    <h1>Collaborate with your team on Test Company</h1>
</div>

custom invite email template